src/main/java/ToDoStoreSingletone.java

```diff
@@ -0,0 +1,74 @@
+import java.util.*;
+
+public class ToDoStoreSingletone implements AddingItem, ChangingStatus, RemovingItem, PrintingItem {
```

Comment:
Can we move all the interfaces with the single method inside to one interface, e.g. 

```
public interface Store {
    public void printAll();
    public void removeItem(int number);
    public void addItem(String s);
    // etc.
}
```